packager/datapackage_descriptor.json

```
@@ -2,69 +2,78 @@
                 {
-                    "name": "month",
+                    "name": "mes",
                     "type": "integer",
                     "format": "default",
-                    "title": "Month",
+                    "title": "Mês",
                     "description": "Month of data collection",
                     "bareNumber": false,
                     "description-ptbr": "O mês que os dados foram coletados",
                     "constraints": {
                         "required": true,
-                        "minimum": "0",
-                        "maximum": "12"
+                        "minimum": 1,
+                        "maximum": 12
                     }
                 },
                 {
-                    "name": "year",
+                    "name": "ano",
                     "type": "integer",
-                    "title": "Year",
+                    "title": "Ano",
                     "description": "Year of data collection",
                     "description-ptbr": "O ano que os dados foram coletados",
                     "constraints": {
                         "required": true,
-                        "minimum": "0"
+                        "minimum": 2010
                     }
                 },
                 {
-                    "name": "reg",
-                    "title": "Register number",
+                    "name": "id_funcionario",
+                    "title": "Número do id dentro do pacote de dados",
+                    "type": "string",
+                    "format": "default",
+                    "description": "Employee registration word",
+                    "description-ptbr": "A palavra que refere-se ao identificador de registro do empregado dentro do pacote de dados"
+                },
+                {
+                    "name": "reg_funcionario",
+                    "title": "Número de registro do funcionário no órgao",
                     "type": "string",
                     "format": "default",
                     "description": "Employee registration word",
                     "description-ptbr": "A palavra que refere-se ao identificador de registro do empregado"
                 },
                 {
-                    "name": "name",
-                    "title": "Name",
+                    "name": "nome_funcionario",
+                    "title": "Nome do Funcionário",
                     "type": "string",
                     "description": "Employee Name",
                     "format": "default",
                     "description-ptbr": "O nome do empregado",
                     "constraints": {
-                        "required": true
+                        "required": true,
+                        "minLength": 2
                     }
                 },
                 {
-                    "name": "role",
-                    "title": "Role",
+                    "name": "cargo",
+                    "title": "Cargo",
```

Para manter consistência:

```suggestion
                    "title": "Cargo do Funcionário",
```